service/grails-app/domain/org/olf/licenses/License.groovy

```diff
@@ -64,4 +64,20 @@ class License implements CustomProperties,MultiTenant<License> {
                links cascade: 'all-delete-orphan'
   }
 
+  public boolean isOpenEnded() {
+    boolean result = false;
+    if ( endDateSemantics?.label?.equalsIgnoreCase('Open Ended') ) 
```

Comment:
This'll get out of date with i18n or custom vocab, should we check the `endDateSemantics.value === 'open_ended'` instead?

Reply:
We probably should, yes. I'll do that.